stable/newrelic-infrastructure/templates/_helpers.tpl

```diff
@@ -20,9 +20,37 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- end -}}
 
 {{/* Generate basic labels */}}
-{{- define "newrelic-infra.labels" }}
-app: {{ template "newrelic-infra.name" . }}
+{{- define "newrelic.labels" }}
+app: {{ template "newrelic.name" . }}
 chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+component: infra
```

Comment:
Now everything gets this component label. I wanted you to remove the label here and add the correct component labels explicitly.

Reply:
that slipped through my refactor sorry about that